qml/pages/VenueMapPage.qml

```diff
@@ -94,6 +94,7 @@ BVApp.Page {
             addMapItem(venueMarker)
             addMapItem(currentPosition)
             center = venueCoordinate
+            fitViewportToMapItems()
```

Comment:
Unfortunately, it still doesn't work on SailfishOS, B/C they did not yet upgrade to QtLocation from Qt 5.2 yet (although Jolla is basically the main contributor…). Well, this is why I forgot to readd it.